components/editor/elements/NoteLinkElement.tsx

```diff
@@ -32,6 +32,7 @@ export default function NoteLinkElement(props: NoteLinkElementProps) {
     <Tooltip content={element.noteTitle} placement="bottom-start">
       <span
         role="button"
+        data-testid={'note-link-element-' + element.noteTitle}
```

Comment:
Instead of adding a `data-testid` here, can we get this element by using `contains`? I want to avoid using the note title in the `data-testid` if possible, because it's user-controlled and could have spaces or be very long.

When using `contains`, the displayed note name could be either the note title or some custom text that the user specifies, but I think that should be okay. We'll just need to account for that when using the command.